### PR TITLE
CRUD operations-for-posts-model

### PR DESCRIPTION
**Pull request by raahimdevsinc**

### CODE REVIEW

**Review by husnain-1125 — COMMENTED** [review]
Update your branch name to be more meaningful (feat/CRUD-operations-for-post-model)

**Comment by devsinc** [comment]
There're numerous offences that can be checked and fixed through robocop, please do that for every of your PR

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def index
 
   # GET /posts/1 or /posts/1.json
   def show
+    return nil
```
Comment (devsinc) [inline app/controllers/posts_controller.rb]: why are we returning nil here?

**Review by devsinc — CHANGES_REQUESTED** [review]

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (raahimdevsinc) [reply app/controllers/posts_controller.rb]: Was trying to figure out how this function was working! Forgot to remove the line in the PR

**Pull request merged**